app/assets/stylesheets/logixRelease.scss

```diff
@@ -91,6 +91,12 @@ a.btn.btn-outline-danger {
   margin-top: 40px;
 }
 
+.profile-image {
```

Comment:
where is this class used? please remove them if unused.

Reply:
Thank you brother ! ❤
- i some hove forgot to add this css class to image 🤦‍♀️ , but it's fixed now